packages/core/src/salt-provider/SaltProvider.tsx

```diff
@@ -435,11 +457,13 @@ interface SaltProviderNextAdditionalProps {
   actionFont?: ActionFont;
 }
 
+/** @deprecated use `SaltProviderProps` with `brand="jpmc"` prop */
 export type SaltProviderNextProps = SaltProviderProps &
   SaltProviderNextAdditionalProps;
-/** @deprecated use `SaltProviderNextProps` */
+/** @deprecated use `SaltProvider` with `brand="jpmc"` prop */
 export type UNSTABLE_SaltProviderNextProps = SaltProviderNextProps;
 
+/** @deprecated use `SaltProvider` with `brand="jpmc"` prop */
```

Comment:
Let's not deprecate this yet, until everything can be proven working. Don't want to cause panic for teams like Ring / Pepper